incubator-tools/language_detection/language_detection.ipynb

```diff
@@ -0,0 +1,535 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        project_id: str = "rand-automl-project",  # "YOUR_PROJECT_ID",
</pre><p>Remove this placeholder project ID</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>